### ISSUE TITLE
Merge, Branch Ideas

### ISSUE THREAD

**Issue by goodnekk** [CodeForEindhoven/IdeaGarden#36]
Some ideas are very similar, some ideas are referencing others. If we allow for this in the interface it would be a really collaborative brainstorm!

**Comment by basbiezemans** [comment]
I like the idea.

**Comment by basbiezemans** [comment]
How about using the references as a measure for idea maturity? Combined with the number of additions and number of participants it can determine the badge stage.

**Comment by goodnekk** [comment]
Yes, maybe we can add the concept of "schools/flocks of fish". Several ideas together can form a flock.

**Comment by basbiezemans** [comment]
Sounds good. How would you visualize the concept?
